base/docs/Docs.jl

```diff
@@ -12,7 +12,7 @@ const modules = Module[]
 
 const META′ = :__META__
 
-meta(mod) = mod.(META′)
+@eval meta(mod) = mod.$META′
```

Comment:
This might actually be more intermittent than I first though it was. The startup freezes are frequent enough that from a usability standpoint it might even be release-blocking. Whatever the issue is appears to be deep in the old LLVM JIT's memory management, as far as we've been able to tell so far.